src/Controls/src/Core/Shell/BaseShellItem.cs

```diff
@@ -511,7 +511,6 @@ internal static DataTemplate CreateDefaultFlyoutItemCell(string textBinding, str
 
 					defaultLabelClass.Setters.Add(new Setter { Property = Label.FontSizeProperty, Value = 14 });
 					defaultLabelClass.Setters.Add(new Setter { Property = Label.TextColorProperty, Value = textColor });
-					defaultLabelClass.Setters.Add(new Setter { Property = Label.FontFamilyProperty, Value = "sans-serif-medium" });
```

Comment:
I think this is the only one that throws from the magic string because the way we load magic strings on `Label` is by consulting the assets first and then it falls back to the `Typeface` call :-/ 

I wonder if there's a way we can change it to not throw so many times or know we need to not consult the assets.